app/helpers/addresses_helper.rb

```diff
@@ -0,0 +1,9 @@
+module AddressesHelper
```

Comment:
it should be placed in `lib` folder or use it as concern https://api.rubyonrails.org/classes/ActiveSupport/Concern.html